### PR TITLE
add cold-staking guides

### PR DESCRIPTION
**Pull request by chasingkirkjufell**

### CODE REVIEW

**Comment by chasingkirkjufell** [comment]
Preview https://deploy-preview-14--navcoin-knowledge-base.netlify.com/general-support/cold-staking/

File: content/General Support/cold staking.en.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Cold-Staking
+date: "2020-02-13"
+draft: false
+---
+
+Cold-staking means that your wallet is not online staking and subject to attacks. Instead, a staking wallet which constains no NAV will be staking on behalf of your spending wallet which holds your NAV. Bear in mind that you still need a nodes actively staking ideally 24/7 to get your staking rewards.
```
Comment (aguycalled) [inline content/General Support/cold staking.en.md]: s/constains/contains
s/nodes/node

File: content/General Support/cold staking.en.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Now that you have spending and staking wallet, you will need to create a cold-staking address which can be created by one spending address and one staking address. You can go to the recieve tab on your core wallet and click the "CREATE A COLD STAKING ADDRESS" button. A window will pop up to guide you through the process. 
+
+Now you can send your NAVs to your cold-staking address. Your spending address from your spending wallet will still hold the NAVs and the power to spend it after you send it.
+
+After you send your coins to the cold-staking address, load your staking wallet with the mahcine you want to stake with (putting the wallet.dat file in the navcoin4 folder will do the trick). After syncing up, your cold-staking balance will be shown. Now turn on staking and enjoy staking with your spending wallet offline. And remember to vote on proposals and consultations with your staking wallet.
```
Comment (aguycalled) [inline content/General Support/cold staking.en.md]: s/mahcine/machine

**Review by aguycalled — CHANGES_REQUESTED** [review]

File: content/General Support/cold staking.en.md (hunk 1 above)
Comment (chasingkirkjufell) [reply content/General Support/cold staking.en.md]: fixed in 50cd644

File: content/General Support/cold staking.en.md (hunk 2 above)
Comment (chasingkirkjufell) [reply content/General Support/cold staking.en.md]: fixed in 50cd644

**Review by aguycalled — APPROVED** [review]

**Pull request merged**